src/installer-win/.gitignore

```diff
@@ -0,0 +1 @@
+bicep/
```

Comment:
I think better to be more precise and prefix with a `/` (or whatever path prefix you're trying to exclude)

Reply:
This .gitignore is specific to this project only, though. I think we did something like that in the VSIX project too.

Reply:
Unless you need to create `bicep` folders at different levels it feels unnecessary. It's often done for more general build artifacts like `out` or `node_modules` which can appear at different levels, but in this case I think you should know exactly where the folder is created.